modules/management/kubectl-apply/variables.tf

```diff
@@ -15,14 +15,24 @@
   */
 
 locals {
-  supported_versions = ["v0.9.1", "v0.9.0", "v0.8.1"]
+  kueue_supported_versions  = ["v0.9.1", "v0.9.0", "v0.8.1"]
+  jobset_supported_versions = ["v0.7.1", "v0.5.2"]
```

Comment:
Would we be deprecating the older jobset version in future? If yes, please create a backlog task.

Reply:
Yes, added the task